Server/controller/deleteBlog.js

```diff
@@ -15,9 +15,9 @@ const deleteBlog = async (req, res) => {
     return res.status(200).json({ message: "successfully deleted" });
   } catch (e) {
     logger.error(e.message);
-    logger.info(JSON.stringify({ "statusCode": 500 }));
+    logger.info(JSON.stringify({ "statusCode": 400 }));
     return res
-      .status(500)
+      .status(400)
```

Comment:
catch block is executed when some error happens, where you have to return Internal server error (500), 400 applies when there is some validation failure.